<launch>	
    <arg name="robot_name" default="default" />

    <param name="robot_description" textfile="test_joint.urdf" />
    <param name="use_gui" value="True"/>
    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" >
        <remap from="joint_states" to="jointstate_cmd" />
    </node>
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find taurob_arm_node)/launch/test.rviz" />

    <node pkg="taurob_arm_node" type="taurob_arm_node" name="taurob_arm_node" output="screen">
        <rosparam file="$(find taurob_arm_node)/config/$(arg robot_name).yaml" command="load"/>
        <remap from="/jointstate_status" to="/joint_states" />
    </node>

    <include file="$(find taurob_watchdog)/launch/watchdog.launch" />
</launch>
